.werft/jobs/build/prepare.ts

```diff
@@ -54,10 +54,11 @@ function configureDocker() {
     }
 }
 
-function configureCoreDevAccess() {
-    const rc = exec('gcloud container clusters get-credentials core-dev --zone europe-west1-b --project gitpod-core-dev', { slice: prepareSlices.CONFIGURE_CORE_DEV }).code;
+function configureStaticClustersAccess() {
+    const rcCoreDev = exec(`KUBECONFIG=${CORE_DEV_KUBECONFIG_PATH} gcloud container clusters get-credentials core-dev --zone europe-west1-b --project gitpod-core-dev`, { slice: prepareSlices.CONFIGURE_CORE_DEV }).code;
+    const rcHarvester = exec(`cp /mnt/secrets/harvester-kubeconfig/harvester-kubeconfig.yml ${HARVESTER_KUBECONFIG_PATH}`, { slice: prepareSlices.CONFIGURE_CORE_DEV }).code;
 
-    if (rc != 0) {
+    if (rcCoreDev != 0 || rcHarvester != 0) {
```

Comment:
I think it would be nice to split this into two separate checks so you can provide a more detailed error message when it fails.

So could be something like

```ts
const rcCoreDev = exec(`KUBECONFIG=${CORE_DEV_KUBECONFIG_PATH} gcloud container clusters get-credentials core-dev --zone europe-west1-b --project gitpod-core-dev`, { slice: prepareSlices.CONFIGURE_CORE_DEV }).code;

if (rcCoreDev != 0) {
  throw new Error("Failed to get core-dev kubeconfig credentials.")
}

const rcHarvester = exec(`cp /mnt/secrets/harvester-kubeconfig/harvester-kubeconfig.yml ${HARVESTER_KUBECONFIG_PATH}`, { slice: prepareSlices.CONFIGURE_CORE_DEV }).code;

if (rcHarvester != 0) {
  throw new Error("Failed to get Harvester kubeconfig credentials.")
}

```